piksi_tools/settings.py

```diff
@@ -330,7 +331,7 @@ def get_args():
         'save', help='save all the current settings to flash.')
 
     reset = subparsers.add_parser(
-        'reset', help='reset the device after the action.')
+        'reset', help='reset the device to default settings.')
```

Comment:
default settings?

Reply:
I think this can be misinterpreted as factory defaults which I think is not the case?

Reply:
it's indeed a reset to factory defaults AFAIK. Clarified.